backend/apiPackage.ts

```diff
@@ -3,6 +3,8 @@ import * as apiSchema from './apiSchema';
 import { Request, Response } from 'express';
```

Comment:
This is in general but maybe see if you can fix the linter issue. It's saying there's a missing npm package for react so see if installing that and repushing a commit fixes the linter issue.

Reply:
Fixed